src/styles/blocks/catalog.scss

```
@@ -16,10 +16,17 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $card-margin;
+
+    transition-duration: 300ms;
```

By default trasition is used for `all`. Need to set what you need
```suggestion
    transition: transform 0.3s, color 0.3s;
```